react/features/settings/components/web/SettingsDialog.js

```diff
@@ -127,9 +127,11 @@ class SettingsDialog extends Component<Props> {
 function _mapStateToProps(state) {
     const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
     const jwt = state['features/base/jwt'];
+    const { prejoinPageEnabled } = state['features/base/config'];
```

Comment:
can you describe `prejoinPageEnabled` somewhere in the UI section of the `config.js` file? https://github.com/jitsi/jitsi-meet/blob/master/config.js#L265